Project3_Bulls_and_Cows/Game.py

```diff
@@ -0,0 +1,75 @@
+import random
+import itertools
+
+def game_header():
+    print('Hi there!')
+    print("I've generated a random 4 digit number for you.")
+    print("Let's play a bulls and cows game.")
+
+
+def number_generator():
+    generated_number = [int(x) for x in range(10)]
+    secret = []
+
+    for i in generated_number:
```

Comment:
What about using condition
```python
while len(secret) < 4:
```
Then you will express the intent and you don't even have to break from the loop. You don't use `i` anyway. And also you should not remove elements from the collection you are looping over.